addons/website/static/src/systray_items/new_content.scss

```diff
@@ -1,80 +1,28 @@
-@keyframes fadeInDownSmall {
-    0% {
-        opacity: 0;
-        transform: translate(0, -5px);
-    }
-    100% {
-        opacity: 1;
-        transform: translate(0, 0);
-    }
-}
+.o_new_content_container {
+    --new-content-item-size: 150px;
 
-#o_new_content_menu_choices {
-    @include o-position-absolute(0, 0, 0, 0);
-    position: fixed;
-    display: flex;
-    overflow: auto;
-    background-color: rgba(0, 0, 0, 0.8);
     font-family: $o-we-font-family;
-    z-index: $zindex-fixed - 1;
 
-    &::before {
-        content: " ";
-        @include o-position-absolute(0, 0, 0, 0);
-        z-index: -1;
-        pointer-events: none;
-    }
+    .popover {
+        max-width: calc(var(--new-content-item-size) * 3 + #{$popover-body-padding-y} * 2 + #{$spacer} * 3);
+        transform: translateX(calc((var(--new-content-item-size) * 3 + #{$popover-body-padding-y} * 2 + #{$spacer} * 2) * -1));
+        top: calc(100% + #{$o-navbar-padding-v} * 2);
```

Comment:
Nitpicking but with this value, the popover is not top-aligned with the dropdowns of the header. That said, I think the problem is only relevant because we are making this popover look like a dropdown. Maybe we could add an arrow pointing to "+ New" button and make it a real popover ?